example/.testcaferc.json

```diff
@@ -1,4 +1,4 @@
 {
-    "appCommand": "yarn start",
-    "appInitDelay": 2000
-}
\ No newline at end of file
+  "appCommand": "yarn start -p 3099",
```

Comment:
Nice!

Reply:
That allows you to run the tests while having another development instance open, right?

Reply:
Yep!